Plugins.md

```diff
@@ -30,6 +30,37 @@ to enable it this can be done via clicking on the green check mark
 
 ![thold plugin enable](images/cacti_thold_enable.JPG)
 
+### Updating a Plugin
+
+Updating a plugin not only keep you up to date with the latest release of the code for that plugin
+it also allows you to keep all of your settings for that plugin ensure you DO NOT UNINSTALL the plugin
+you must instead disable the plugin which will not drop the SQL tables and thus save your settings
```

Comment:
There are some grammatical issues here (one very long sentence). Also, `keep` on the first line should really be `keeps`.